build.gradle

```
@@ -28,8 +28,8 @@ buildscript {
 
     'googleMaterial': 'com.google.android.material:material:1.0.0',
 
-    'rxJava': 'io.reactivex.rxjava2:rxjava:2.2.10',
-    'rxAndroid': 'io.reactivex.rxjava2:rxandroid:2.1.1',
+    'rxJava': 'io.reactivex.rxjava3:rxjava:3.0.0',
```

Might as well pick up the latest patch

```suggestion
    'rxJava': 'io.reactivex.rxjava3:rxjava:3.0.2',
```